news/2024/2024-06-03-the-followpoint-eriha-top-drum-basher.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "The Followpoint: Eriha, Top Drum Basher"
+date: 2024-06-03 11:00:00 +0000
+---
+
+It's the latest in a series of top player interviews, now featuring the osu!taiko #4 best drum basher: [Eriha](https://osu.ppy.sh/users/16320311)!
+
+![](/wiki/shared/news/2024-06-03-the-followpoint-eriha-top-drum-basher/banner.jpg)
+
+That's right, following previous interviews with [mrekk](https://osu.ppy.sh/home/news/2024-05-06-the-followpoint-mrekk-the-osu-standard-prodigy) and [YesMyDarknesss](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch), top players of osu! and osu!catch respectively, this edition of The Followpoint continues the pattern by featuring osu!taiko's rank #4 player and the [osu!taiko World Cup 2024](/wiki/Tournaments/TWC/2024) champion.
+
+Simply read on for a look into [Eriha](https://osu.ppy.sh/users/16320311)'s insight:
+
+---
+
+*Let's start with an introduction, how would you introduce yourself to people who don't know you?*
+
+> Hello! My name is Eriha and I'm Japanese. I'm 18 years old. My current global ranking is #4, which is also my peak rank.
+>
+> I participated in the recent TWC2024 as a member of the Japanese team and became the osu!taiko Champion. Many of you know me due to my PP Record during the previous SR system!
+>
+> I'm currently working with Project Loved Captain and was a Beatmap Nominator for a year in 2022-2023. I'm not positive about returning to BN activities due to my busy college schedule, but I would like to have a job in the future where I can support the mapper's activities.
+
+*How did you get introduced to osu! and what was your first impression of it?*
+
+> I found osu! 4 years ago when a std player friend recommended it to me. I have been playing Taiko no Tatsujin since I was very young so my friend told me that taiko mode is the best for me.
```

Comment:
```suggestion
> I found osu! 4 years ago when a std player friend recommended it to me. I have been playing Taiko no Tatsujin since I was very young, so my friend told me that taiko mode is the best for me.
```